deploy/nginx.conf

```diff
@@ -0,0 +1,24 @@
+server {
+  listen 8080;
+  server_name localhost;
+
+  root $NGINX_APP_ROOT/src/;
+  index index.html;
+
+  location /api {
+      proxy_pass $ASSISTED_SERVICE_URL;
```

Comment:
The value here may differ from the value in the deployment template, is that OK?